### PR TITLE
Lesson 1_3 Props State and Handlers

### PR DESCRIPTION
**Pull request by Samelo01**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
-  { id: 1, title: "Complete assignment" },
-  { id: 2, title: "Review code" },
-  { id: 3, title: "Submit project" }
-];
+const App = () => {
+    const [todos, setTodos] = useState([]);
+    const [newTodo, setNewTodo] = useState('');
+
+    const handleAddTodo = (todoTitle) => {
+        setTodos([...todos, { id: Date.now(), title: todoTitle }]);
```
Comment (justiceotuya) [inline src/App.js:11]: based on the lesson for this week, this might not be needed

File: src/App.js [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
+        setNewTodo(todoTitle);
+    };
+
+    return (
+        <div>
+            <AddTodoForm onAddTodo={handleAddTodo} />
+            <p>New Todo: {newTodo}</p>
+            <TodoList todos={todos} />
```
Comment (justiceotuya) [inline src/App.js:19]: remember that the todos you are passing here is not been used on the todolist

File: src/TodoList.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// src/TodoList.jsx
+import React from 'react';
+import TodoListItem from './TodoListItem';  // Import TodoListItem
+
+function TodoList() {
```
Comment (justiceotuya) [inline src/TodoList.jsx:5]: Keep this in mind that youve passed a `todolist` but you are not using it here

**Review by justiceotuya — APPROVED** [review]
I will approve this because it works and you ticked off all the boxes in this week tasks.

You are ahead of the class and you should be proud

**Comment by Samelo01** [comment]
Thank you so much for your time and help

**Pull request merged**